package.json

```diff
@@ -55,8 +55,9 @@
     }
   },
   "scripts": {
-    "build": "grunt",
-    "test": "grunt test",
+		"build": "grunt",
+		"test-dts": "tsc --noImplicitAny typings/axe-core/axe-core-tests.ts",
```

Comment:
We should disable other language features too (implicit this, etc).

Also, these options are better stored in `tsconfig.json`, as the CLI flags will get out of hand quickly.

Reply:
agreed, moved to tsconfig.json